packages/flutter/lib/src/widgets/icon_theme_data.dart

```diff
@@ -87,6 +90,9 @@ class IconThemeData with Diagnosticable {
   /// The default size for icons.
   final double? size;
 
+  /// The default splash radius for [IconButton]s.
+  final double? splashRadius;
```

Comment:
This doesn't really belong in a widgets library class, IconButton is from the Material library and ink splashes are a feature of Material.

Reply:
Can I revert this PR?

Reply:
Yes. We should talk over how to handle this issue in the material library

Reply:
Revert https://github.com/flutter/flutter/pull/94482